apps/cowswap-frontend/src/modules/ordersTable/pure/TwapOrderStatus/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { ReactNode } from 'react'
```

Comment:
😄 

Sorry, React imports are everywhere

Reply:
I will kill them all later :) 
And I will add a prettier rule